Environmental/DRAXIS/Python/requirements.txt

```diff
@@ -0,0 +1,14 @@
+certifi==2019.11.28
```

Comment:
maybe it is better to list only the explicit requirements eg elasticsearch, kafka-python etc and not their dependencies?